specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/communityGallery.json

```diff
@@ -0,0 +1,609 @@
+        "parameters": [
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./galleryRPCommon.json#/parameters/LocationNameParameter"
+          },
+          {
+            "$ref": "../../../common-types/v1/common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/PublicGalleryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/GalleryImageNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CommunityGalleryImageVersionList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common-types/v1/common.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List community gallery image versions.": {
+            "$ref": "./examples/communityGalleryExamples/CommunityGalleryImageVersion_List.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PirCommunityGalleryResource": {
+      "properties": {
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name"
+        },
+        "location": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource location"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type"
+        },
+        "identifier": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CommunityGalleryIdentifier"
+        }
+      },
+      "description": "Base information about the community gallery resource in pir."
+    },
+    "CommunityGalleryArtifactTags": {
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
+      },
+      "description": "The artifact tags of a community gallery resource."
+    },
+    "CommunityGalleryDisclaimer": {
+      "type": "string",
+      "properties": {
+        "disclaimer": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Community gallery disclaimer"
+        }
+      },
+      "description": "The disclaimer for a community gallery resource."
+    },
+    "CommunityGalleryIdentifier": {
+      "properties": {
+        "uniqueId": {
+          "type": "string",
+          "description": "The unique id of this community gallery."
+        }
+      },
+      "description": "The identifier information of community gallery."
+    },
+    "CommunityGalleryMetadata": {
+      "type": "object",
+      "properties": {
+        "publisherUri": {
```

Comment:
>publisherUri [](http://example.com/codeflow?start=9&length=12)

Consider adding format: uri

Reply:
added in https://github.com/Azure/azure-rest-api-specs/pull/25794